### ISSUE TITLE
Accessing mockbot token API blocked by CORS policy in "clear-after-idle" sample

### ISSUE THREAD

**Issue by stevkan** [microsoft/BotFramework-WebChat#3910]
## Version
v4.13.0

NPM package

## Describe the bug
Running the [h.clear-after-idle](https://github.com/microsoft/BotFramework-WebChat/tree/main/samples/04.api/h.clear-after-idle) sample generates an error when the host application attempts to access mockbot's token API. The host application is blocked do to CORS policy.

![image](https://user-images.githubusercontent.com/18403946/119401091-e1d35200-bc8f-11eb-8d98-6c5abae7e3ca.png)

## Steps to reproduce

1. Clone repo and install packages
2. Run sample
3. Observe error in browser

## Expected behavior
The host app should be able to request a token from mockbot's API.

**Comment by corinagum** [comment]
@compulim I don't think that format of IP address is added to `trustedOrigins` in MockBot. Are you interested in adding it?
